### PR TITLE
Added multiple tests for Iteration 2 

### PR DESCRIPTION
**Pull request by chrisstankus99**
Tests added: 
valid coordinates 
valid placement 
valid consecutive placement 
valid diagonal placement

### CODE REVIEW

File: test/board_test.rb [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  def test_board_can_check_valid_coordinate
+    assert @board.valid_coordinate("A1")
+    assert @board.valid_coordinate?("D4")
+    refute @board.valid_coordinate?("A5")
+    refute @board.valid_coordinate?("E1")
+    refute @board.valid_coordinate?("A22")
+  end
+
+  def test_board_can_check_valid_placement
+  refute @board.valid_placement?(cruiser, ["A1", "A2"])
```
Comment (tylertomlinson) [inline test/board_test.rb]: Indentation

File: test/board_test.rb [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  refute @board.valid_placement?(submarine, ["A2", "A3", "A4"])
+  assert @board.valid_placement?(submarine, ["A1", "A2"])
+  assert @board.valid_placement?(cruiser, ["B1", "C1", "D1"])
+  end
+
+  def test_board_can_check_if_valid_placement_is_consecutive
+  refute @board.valid_placement?(cruiser, ["A1", "A2", "A4"])
```
Comment (tylertomlinson) [inline test/board_test.rb]: Indentation

File: test/board_test.rb [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+  refute @board.valid_placement?(submarine, ["A1", "C1"])
+  refute @board.valid_placement?(cruiser, ["A3", "A2", "A1"])
+  refute @board.valid_placement?(submarine, ["C1", "B1"])
+  end
+
+  def test_board_can_check_valid_placement_is_not_diagonal
+  refute @board.valid_placement?(cruiser, ["A1", "B2", "C3"])
```
Comment (tylertomlinson) [inline test/board_test.rb]: Indentation

File: test/board_test.rb (hunk 1 above)
Comment (chrisstankus99) [reply test/board_test.rb]: Fixed indentation

**Comment by tylertomlinson** [comment]
Looks good! Might need to add more tests and also break some of these up to make more robust. Going to merge then pull into my branch to run.

File: test/board_test.rb (hunk 2 above)
Comment (chrisstankus99) [reply test/board_test.rb]: fixed indentation

File: test/board_test.rb (hunk 3 above)
Comment (chrisstankus99) [reply test/board_test.rb]: fixed indentation

**Pull request merged**